evaluator/evaluator.py

```diff
@@ -338,16 +339,20 @@ def process_message(message):
                 msg_dict = json.loads(message.value.decode('utf-8'))
                 if 'type' not in msg_dict:
                     LOGGER.error("Received message is missing type field: %s", msg_dict)
+                    send_msg_to_payload_tracker(msg_dict, 'error', status_msg="Received message is missing type field: %s" % msg_dict, loop=loop)
                     return
                 if msg_dict['type'] in ['upload_new_file', 're-evaluate_system']:
                     process_func = self.process_upload_or_re_evaluate
+                    send_msg_to_payload_tracker(msg_dict, 'processing', loop=loop)
```

Comment:
this should be logged only for message of type 'upload_new_file', 're-evaluate_system' is not payload message